Data/Source/Scripts/AutoTest_Suite_NPCs.psc

```diff
@@ -74,20 +81,38 @@ endFunction
 ; * *baseId* (Integer): The BaseID to clone and take screenshot
 ; * *espName* (String): The name of the ESP containing this base ID
 function ScreenshotOf(int baseId, string espName)
+
   int formId = baseId + Game.GetModByName(espName) * 16777216
   Form formToSpawn = Game.GetFormFromFile(formId, espName)
   string formName = formToSpawn.GetName()
   Log("[ " + espName + "/" + baseId + " ] - [ Start ] - Take screenshot of FormID 0x" + formId + " (" + formName + ")")
   Game.GetPlayer().MoveTo(ViewPointAnchor)
   ObjectReference newRef = TeleportAnchor.PlaceAtMe(formToSpawn)
-  newRef.RemoveAllItems()
+  ; TODO: Add option for clothes here
+
+  string nonNudeNPC = GetConfig("non_nude")
+  if nonNudeNPC != "true"
+    nonNudeNPC = "false"
+  endIf
+
+  if nonNudeNPC == "false"
+    newRef.RemoveAllItems()

```

Comment:
I think we can put the `newRef.RemoveAllItems()` directly in the previous `if`, without explicitely setting `nonNudeNPC` to `false`.

Reply:
sounds good